cnf-certification-test/preflight/suite.go

```diff
@@ -19,55 +19,79 @@ package preflight
 import (
 	"strings"
 
-	"github.com/onsi/ginkgo/v2"
 	plibRuntime "github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
 	"github.com/sirupsen/logrus"
 	"github.com/test-network-function/cnf-certification-test/cnf-certification-test/common"
 	"github.com/test-network-function/cnf-certification-test/cnf-certification-test/identifiers"
+	"github.com/test-network-function/cnf-certification-test/pkg/checksdb"
+	"github.com/test-network-function/cnf-certification-test/pkg/configuration"
 	"github.com/test-network-function/cnf-certification-test/pkg/provider"
 	"github.com/test-network-function/cnf-certification-test/pkg/testhelper"
 	"github.com/test-network-function/cnf-certification-test/pkg/tnf"
 )
 
-var _ = ginkgo.Describe(common.PreflightTestKey, func() {
-	logrus.Debugf("Entering %s suite", common.PreflightTestKey)
-	env := provider.GetTestEnvironment()
-	ginkgo.BeforeEach(func() {
+var (
+	env provider.TestEnvironment
+
+	beforeEachFn = func(check *checksdb.Check) error {
+		logrus.Infof("Check %s: getting test environment.", check.ID)
 		env = provider.GetTestEnvironment()
-	})
+		return nil
+	}
+)
 
-	// Add safeguard against running the tests if the docker config does not exist.
-	if env.GetDockerConfigFile() == "" || env.GetDockerConfigFile() == "NA" {
-		logrus.Debug("Skipping the preflight suite because the Docker Config file is not provided.")
-		return
+func labelsAllowTestRun(labelFilter string, allowedLabels []string) bool {
```

Comment:
This could be stringhelper.StringInSlice.

Reply:
🤔 They look pretty similar to me unless I'm missing something? 

https://github.com/test-network-function/cnf-certification-test/blob/main/pkg/stringhelper/stringhelper.go#L24C66-L24C66

You can pass the contains boolean that does the `strings.Contains` as well.